ui/component/publishSettings/view.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import { FormField } from 'component/common/form';
+import { withRouter } from 'react-router';
+
+type Props = {
+  enablePublishPreview: boolean,
+  setEnablePublishPreview: boolean => void,
+};
+
+function PublishSettings(props: Props) {
+  const { enablePublishPreview, setEnablePublishPreview } = props;
+
+  function handleChange() {
+    setEnablePublishPreview(!enablePublishPreview);
+  }
+
+  return (
+    <div>
+      <FormField
+        type="checkbox"
+        name="sync_toggle"
+        label={__('Review changes before uploading.')}
```

Comment:
this text should match or be similar to the text that allows toggling it at the bottom of the confirmation dialog